config/api.yml

```diff
@@ -477,10 +477,17 @@
       :post:
       - :name: query
         :identifier: cloud_volume_show_list
+      - :name: delete
```

Comment:
@AparnaKarve I think I confused myself :) This is correct - it's the one below that's not covered by a test